src/lib/features/metrics/instance/findOutdatedSdks.ts

```diff
@@ -14,16 +14,18 @@ const config: SDKConfig = {
     'unleash-client-php': '1.13.0',
 };
 
+export const isOutdatedSdk = (sdkVersion: string) => {
```

Comment:
usually our files are not camel case

Reply:
this is a good point. I usually use - when we use classes and camel case when the file only has functions

Reply:
but maybe we should move all files in backend code to use dashes